packages/react/src/auto/mui/inputs/MUIDateTimePicker.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box } from "@mui/material";
+import { DatePicker } from "@mui/x-date-pickers";
+import { TimePicker } from "@mui/x-date-pickers/TimePicker";
+import React from "react";
+import { useFieldMetadata } from "../../hooks/useFieldMetadata.js";
+
+export const MUIDateTimePicker = (props: { field: string; value?: Date; onChange: (value: Date) => void; error?: string }) => {
+  const { metadata } = useFieldMetadata(props.field);
+
+  return (
+    <Box sx={{ display: "flex" }}>
+      <DatePicker label={metadata.name} />
+      <TimePicker />
```

Comment:
These 2 would not be connected to the submit button. They need a `useController` to get the corresponding `value` and `onChange` values that are connected to the form context